src/tree/transform.js

```diff
@@ -116,15 +168,15 @@ export function setZooming({ objectData, setTransform, transformState, selection
         () =>
           !selectionsAndTransform.constraints.active &&
           event.type !== 'dblclick' &&
-          !(event.type === 'mousedown' && event.which === 3)
+          !(event.type === 'mousedown' && event.which === 3) &&
+          event.type !== 'touchstart'
```

Comment:
Should be able to only check for event.type === "mousewheel" or something.

This should be ok to keep, saves us some trouble with implementing the correct scroll/pos logic.

Reply:
I might need to rename this. This is also panning for desktop. For D3 these need to be combined otherwise x and y are not in sync